inference-engine/ie_bridges/python/tests/test_IECore.py

```diff
@@ -49,6 +49,7 @@ def test_load_network_wrong_device():
     assert 'Device with "BLA" name is not registered in the InferenceEngine' in str(e.value)
 
 
+@pytest.mark.skip(reason="IENetwork.layers return not all layers in case of ngraph representation due to inner conversion into legacy representation")
```

Comment:
Why did you skip these tests for all plugins?
When will it be enabled?

Reply:
Python API Converts to old representation internally. There should be separate tests for legacy and ngraph representation or there should be no conversion in API.

Reply:
Can you create a ticket for this activity?